ts/navigation/AppStackNavigator.tsx

```diff
@@ -172,19 +173,22 @@ export const AppStackNavigator = () => {
         <Stack.Screen name={FCI_ROUTES.MAIN} component={FciStackNavigator} />
       )}
 
-      <Stack.Screen
-        name={IDPayOnboardingRoutes.IDPAY_ONBOARDING_MAIN}
-        component={IDPayOnboardingNavigator}
-      />
-      <Stack.Screen
-        name={ROUTES.IDPAY_INITIATIVE_DETAILS}
-        component={InitiativeDetailsScreen}
-      />
-
-      <Stack.Screen
-        name={IDPayConfigurationRoutes.IDPAY_CONFIGURATION_MAIN}
-        component={IDPayConfigurationNavigator}
-      />
+      {idPayEnabled && (
+        <>
+          <Stack.Screen
+            name={IDPayOnboardingRoutes.IDPAY_ONBOARDING_MAIN}
+            component={IDPayOnboardingNavigator}
+          />
+          <Stack.Screen
+            name={ROUTES.IDPAY_INITIATIVE_DETAILS}
+            component={InitiativeDetailsScreen}
+          />
+          <Stack.Screen
+            name={IDPayConfigurationRoutes.IDPAY_CONFIGURATION_MAIN}
+            component={IDPayConfigurationNavigator}
+          />
+        </>
+      )}
```

Comment:
Just for curiosity... Which is better as approach? Using a <> </> OR split them in single Stack.Screen with conditional? Like `CgnEnabled` in line 133..135

Reply:
I think that using a Fragment is less error prone than splitting each component with its conditional.
The `idPayEnabled` flag enables/disables all idPay routes at once, there is no need to repeat the same thing for each `Stack.Screen`, imho :)